docs/howtos/convert_pytorch_to_flax.rst

```diff
@@ -0,0 +1,276 @@
+Convert PyTorch Models to Flax
+==============================
+
+.. testsetup::
+
+  import numpy as np
+  import jax
+  from jax import random, numpy as jnp
+  import flax
+
+  from flax import linen as nn
+  from flax.core import freeze
+
+  import torch
```

Comment:
this is currently failing because `torch` is not listed as a `[testing]` dependency.

could you add it to to `setup.py` in the same PR?
(we'd have to look at overall check setup and run time to see if this causes a significant change before deciding if we want to add it for real)

Reply:
Note: adding `torch` adds about 30 seconds to the `Install Dependencies` step in the build. The build time variance is rather large (12 min ... 20 min)